### ISSUE TITLE
2.6-beta: Shapeless matrix delimiter in CommonHTML

### ISSUE THREAD

**Issue by mizar** [mathjax/MathJax#1289]
sample: https://gist.github.com/anonymous/7c6eeab3a19ebd842d83

Font outside the block seems to affect the shape loss of matrix delimiter.

**Comment by szego** [comment]
Changing "config=TeX-AMS_CHTML" to "config=TeX-AMS_HTML-full" when importing MathJax.js seems to fix it for me. Does it fix things on your end too?

**Comment by mizar** [comment]
There are no problems other than the performance in HTML-CSS mode rendering.
Is it intended that there is no need to fix this issue of CommonHTML mode?

**Comment by pkra** [comment]
Thanks for reporting this! It's definitely a bug and should be fixed.

**Comment by dpvc** [comment]
I believe that the fix for issue #1326 will resolve this problem.

**Comment by mizar** [comment]
http://jsbin.com/cedazewari/2
https://gist.github.com/anonymous/58e8ffee618efe8d9516

Perhaps it is still unresolved.

**Comment by dpvc** [comment]
The fix hasn't been pushed to the beta.mathjax.org site yet, so your examples aren't using the new code.  You would have to check out a copy from github yourself to test this.

I looked at your example after making the change in issue 1326, and I did note that there is a race condition between MathJax and the web fonts that you load, so the result will depend on whether MathJax runs before or after the web fonts arrive.  You may want to use the MatchWebFonts extension to make sure the result is properly sized for the fonts once they do arrive, or you could configure MathJax to skip its startup typesetting pass, then wait for the fonts to arrive and tell MathJax to typeset at that point.  (I think the google fonts have an API for detecting their arrival.)

**Comment by mizar** [comment]
Ah, I mistaken development cdn url.
I was confirmed to resolve the problem. Deal and advice Thank you.

http://jsbin.com/sawiwixuho/1
https://gist.github.com/anonymous/72211da5de70fcb6eed1

**Comment by pkra** [comment]
> the fix hasn't been pushed to the beta.mathjax.org site yet, 

There's actually a `beta.mathjax.org/mathjax/develop/` now which should be up to date.

**Comment by pkra** [comment]
Addendum: only `beta.mathjax.org/mathjax/develop/unpacked` would be change since we don't pack except for releases.

**Comment by dpvc** [comment]
If I understand @mizar correctly, he confirms the fix, so I'm closing the issue.
